### PR TITLE
Adds and Exports Icons

### PR DESCRIPTION
**Pull request by flygerian**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -20,7 +21,8 @@ void main() {
           Buttons(),
           AppCards(),
           Popover(),
-          Layouts()
+          Layouts(),
+          IconsStory()
```
Comment (almeynman) [inline lib/main.dart]: mb just Icons for consistency, as this classname is used for displaying a list item in the first screen

File: lib/widgets/change_icon/change_icon.dart [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    return ChangeIconState(_iconData);
+  }
+}
+
+
+class ChangeIconState extends State<ChangeIcon> {
+
+  ChangeIconData _iconData;
+  String _svgContents;
+  final bool isStandAloneMode;
```
Comment (almeynman) [inline lib/widgets/change_icon/change_icon.dart]: do we still need this?

File: android/local.properties [hunk 3]
```diff
@@ -0,0 +1,3 @@
+sdk.dir=/Users/olabodeadedoyin/Library/Android/sdk
```
[the remark this answers is not in the document]
Comment (flygerian) [reply android/local.properties]: Fixed

File: lib/main.dart (hunk 1 above)
Comment (flygerian) [reply lib/main.dart]: Fixed. I called it Iconography so it does not clash with flutter's native Icons class

File: lib/widgets/change_icon/change_icon.dart (hunk 2 above)
Comment (flygerian) [reply lib/widgets/change_icon/change_icon.dart]: Nope! I actually thought I deleted that

**Pull request merged**